raml/response_examples/200/runbook_runs.json

```diff
@@ -0,0 +1,84 @@
+{
+	"runs": [
+		{
+			"_id": "434",
+			"finished_at": "2016-12-20T16:27:42.150227",
+			"output": "SGVsbG8gV29ybGQhCg==",
+			"return_code": 0,
+			"started_at": "2016-12-20T16:18:42.150142",
```

Comment:
This time format differs of time that comes from /api/v1/status/availability/day, for example: "2016-12-22T00:00"
We need to use the same format.